### ISSUE TITLE
No module named 'magic' 

### ISSUE THREAD

**Issue by kaikiat** [ahupp/python-magic#264]
I am getting this error when I try to use Docker + python-magic in my project.

<img width="727" alt="Screenshot 2022-04-08 at 7 25 51 PM" src="https://user-images.githubusercontent.com/32543853/162426632-97fe560b-08fa-45d6-823d-2815bfe2ff8c.png">

This is my Dockerfile. I have tried installing `libmagic-dev` and `libmagic1libmagic1` and ` python3-magic` but it does not seem to work.
```
FROM python:3.9

ADD . /

ENV TZ=Asia/Singapore

ARG CHROME_VERSION="99.0.4844.84-1"

RUN apt-get -y update
RUN wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb \
  && apt install -y /tmp/chrome.deb \
  && rm /tmp/chrome.deb
RUN apt-get install -y tzdata 

RUN apt-get install -y libmagic-dev
RUN apt-get install -y libmagic1
RUN apt-get install python3-magic

WORKDIR /code

COPY . /code
RUN pip install -r /code/requirements.txt

CMD exec gunicorn --bind 0.0.0.0:5000 --workers 1 --worker-class uvicorn.workers.UvicornWorker --threads 2 app.main:app
```

And this is a portion of my requirements.txt
<img width="642" alt="Screenshot 2022-04-08 at 7 29 11 PM" src="https://user-images.githubusercontent.com/32543853/162427064-2090b58e-f818-4b3c-936a-359ff423ab2f.png">

I am not sure if you have knowledge on this and any help will be greatly appreciated. Thank you!

**Comment by ahupp** [comment]
The python3.9 docker image ships its build of python in /usr/local/bin/python3.  This is independent of the underlying distribution's copy of python in /usr/bin/python.  When you install python3-magic it goes into /usr/lib/python3/dist-packages/ and is only visible to the distribution install.  e.g, you can run  /usr/bin/python3 and then `import magic` works.


The second issue is that your requirements.txt is specifying platform_system requirements that will exclude this docker image (which is linux).  The third issue is that python-magic-bin is a fork of python-magic that includes binaries, but it's not the one in this repo.

So tl;dr you should:

1. Only `apt install libmagic1` in the container, you don't need libmagic1-dev or python3-magic
2. Remove the `platform_system` requirements in your requirements.txt
3. Put python-magic, not python-magic-bin in your requirements.txt

**Comment by kaikiat** [comment]
Thanks it works as expected !
